src/Tasks/SearchReindex.php

```diff
@@ -40,18 +43,15 @@ public function __construct(
         $this->setBatchProcessor($batchProcessor);
     }
 
-    /**
-     * @param HTTPRequest $request
-     */
-    public function run($request): void // phpcs:ignore SlevomatCodingStandard.TypeHints
+    protected function execute(InputInterface $input, PolyOutput $output): int
```

Comment:
might need to separate out the job queuing function to be compatible with https://github.com/silverstripeltd/silverstripe-forager/pull/29

Reply:
Sorry, could you expand on that a bit? I'm not sure I understand.

Reply:
well the changes in #29 mean the FormActions in the admin call the job 'run' method. I was thinking it might be an idea to create a method separate from the build task stuff around input/output or we'd need to also implement that in the FormActions.